doc/source/cluster/kubernetes/user-guides/azure-aks-gpu-cluster.md

```diff
@@ -0,0 +1,51 @@
+(kuberay-aks-gpu-cluster-setup)=
+
+# Start Azure AKS Cluster with GPUs for KubeRay
+
+This guide walks you through the steps to create an Azure AKS cluster with GPU nodes specifically for KubeRay.
+The configuration outlined here can be applied to most KubeRay examples found in the documentation.
+
+## Step 1: Create a Resource Group
+
+To create a resource group in a particular region:
+
+```
+az group create -l eastus -n kuberay-rg
+```
+
+## Step 2: Create AKS Clusterhelm uninstall raycluster
+
+To create an AKS cluster with system nodepool:
+```
+az aks create \
+   -g kuberay-rg \
+   -n kuberay-gpu-cluster \
+   --nodepool-name system \
+   --node-vm-size Standard_D8s_v3 \
+   --node-count 3
+```
+
+## Step 3: Add a GPU node group
+
+To add a GPU nodepool with autoscaling:
+```
+az aks nodepool add \
+   -g kuberay-rg \
+   --cluster-name kuberay-gpu-cluster \
+   --nodepool-name gpupool \
+   --node-vm-size Standard_NC6s_v3 \
+   --node-taints nvidia.com/gpu=present:NoSchedule \
+   --min-count 1 \
```

Comment:
Do AKS nodepools support scale-from-zero? If so, we may want to recommend `min-count=0` to keep from wasting resources when not in use.
```suggestion
   --min-count 0 \
```

Reply:
Fixed, AKS supports 0 nodes for user node pool, but not system node pool